sass/_colors.scss

```diff
@@ -1,7 +1,8 @@
-$background:#E5E5E5;
-$main-font-color:#000000;
-$header-bgc:#3CA89A;
-$gif-bg:#F1F1F1;
-$btn-color:#43B9AA;
-$pure-white:#ffffff;
-$line-color:#4A4A4A;
\ No newline at end of file
+$background: #e5e5e5;
+$main-font-color: #000000;
+$header-bgc: #3ca89a;
+$gif-bg: #f1f1f1;
+$btn-color: #43b9aa;
+$pure-white: #ffffff;
+$line-color: #4a4a4a;
+$image-bg: #c4c4c4;
```

Comment:
Czy stosujesz jakąś konkretną metodologię nazewnictwa tych zmiennych, czy tak po prostu wyszło chaotycznie? :)

Reply:
Przyznam, że wygląda to dosyć specyficznie, ja się odnajduję ale po namyśle doszedłem do wniosku, że jeżeli ktoś by chciał coś zmieniać to mógłby nie skumać o co chodzi :) Przy następnym projekcie postaram się nazywać zmienne w bardziej zrozumiały i mniej chaotyczny sposób.